---
uint32 nActiveEngineID
string strActiveEngineDescription
int32 nRetCode
